doc/nrf/releases_and_maturity/migration/2.4.99-cs3_to_2.6.99-cs2/migration_guide_2.4.99-cs3_to_2.6.99-cs2_environment.rst

```diff
@@ -0,0 +1,371 @@
+.. _migration_cs3_to_2_6_99_cs2_env:
+
+Update your development environment for |NCS| v2.6.99_cs2 (for v2.4.99-cs3 users)
+#################################################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This document describes how to update your development environment from |NCS| v2.4.99-cs3 to |NCS| v2.6.99-cs2.
+
+The main development environment changes introduced by 2.6.99-cs2 for the nRF54H20 DK are the following:
+
+* The |NCS| toolchain has been updated.
+  A bootstrap script specific to the nRF54H20 DK is now available to install and update the toolchain.
+* nRF Util has now replaced nRF Command Line Tools.
+  The bootstrap script will now install and update nRF Util.
+* SDFW and SCFW are now provided as precompiled binaries.
+  The Secure Domain Firmware (SDFW) and System Controller Firmware (SCFW) are no longer built from the source during the application build process, but they are installed as binaries from the provided firmware bundle and programmed when flashing the application to the device.
```

Comment:
They are not programmed during flashing application. They need to be programmed by users before DK can be used but won't be preprogrammed on DK.